packages/react/src/components/table/table.tsx

```diff
@@ -170,10 +130,83 @@ const StyledTFoot = styled.tfoot`
     background: inherit;
 `;
 
+const ExpandButton = styled(IconButton)<{ $expanded: boolean }>`
+    transform: rotate(${({ $expanded }) => ($expanded ? 90 : 0)}deg);
+    transition: transform 0.2s ease-in-out;
+
+    &[aria-expanded='true'] {
+        background-color: transparent;
+    }
+`;
+
+function getUtilityColumn<T extends object>(type: string, t: TFunction<'translation'>): TableColumn<T> {
+    return {
+        id: type,
+        className: utilColumnClassName,
+        header(props: HeaderContext<T, unknown>) {
+            if (type === 'selection') {
+                const { table } = props;
+                return (
+                    <Checkbox
+                        data-testid="row-checkbox-all"
+                        checked={table.getIsAllRowsSelected()}
+                        indeterminate={table.getIsSomeRowsSelected()}
+                        onChange={table.getToggleAllRowsSelectedHandler()}
+                    />
+                );
+            }
+            // For 'numbers' type or any other type, return null or an empty header
+            return null;
+        },
+        cell({ row }) {
+            if (type === 'selection') {
+                return (
+                    <Checkbox
+                        data-testid={`row-checkbox-${row.index}`}
+                        checked={row.getIsSelected()}
+                        disabled={!row.getCanSelect()}
+                        indeterminate={row.getIsSomeSelected()}
+                        onChange={row.getToggleSelectedHandler()}
+                    />
+                );
+            }
+
+            if (type === 'numbers') {
+                return (
+                    <RowNumber>{row.index + 1}</RowNumber>
+                );
+            }
+
+            if (type === 'expand' && row.getCanExpand()) {
+                const isExpanded = row.getIsExpanded();
+                return (
+                    <ExpandButton
+                        type="button"
+                        buttonType="tertiary"
+                        iconName="caretRight"
+                        onClick={row.getToggleExpandedHandler()}
+                        aria-expanded={isExpanded}
+                        $expanded={isExpanded}
+                        aria-label={
+                            row.subRows.length > 0
+                                ? t('subrowsAriaLabel', { count: row.subRows.length })
```

Comment:
Présentement l'`aria-label` est `{{count}} subrows for this item`.
Par contre le `this item` est trop vague.
Est-ce que le `this item` pourrait être spécifique à la row? 
Genre associer le content d'un `td` ou d'un `th scope="row"` d'une row?
`{{count}} subrows for [id ou whatever du row header]`

Reply:
Apres discussion entre nous, on s'est convenu de laisser tel quel pour le moment car c'est pas si evident que ca.